src/SprykerEco/Yves/Computop/Mapper/Init/PrePlace/CreditCardMapper.php

```diff
@@ -66,4 +66,25 @@ protected function getActionUrl()
     {
         return $this->config->getCreditCardInitAction();
     }
+
+    /**
+     * @param string $merchantId
+     * @param string $data
+     * @param int $length
+     *
+     * @return string
+     */
+    protected function getUrlToComputop($merchantId, $data, $length)
```

Comment:
This method is too similar to AbstractPrePlaceMapper->getUrlToComputop();
Seems to me that they could be combined into a service class that they both use.
 ComputopApiConfig::URL_BACK could be an optional argument.